src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLInputObjectType,
+  GraphQLObjectType,
+  GraphQLList,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { SmsSecondFactor } from "../secondFactors"
+import { ResolverContext } from "types/graphql"
+
+const SmsSecondFactorAttributes = new GraphQLInputObjectType({
+  name: "SmsSecondFactorAttributes",
+  fields: {
+    countryCode: {
+      type: GraphQLString,
+    },
+    phoneNumber: {
+      type: GraphQLString,
+    },
+  },
+})
+
+const ErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Error",
```

Comment:
This is interesting...basically nothing complains that this is a duplicate type, schema generation and resolution looks correct - even though this type is defined in Gravity as well.

This is necessary b/c clients do `... on Errors` in the second factor mutation.

This is a bit weird, but I'll keep this pattern going in an effort to unstitch with a no-op.

Reply:
Super weird, but sweet 😎